src/lib/ical/parse.test.ts

```diff
@@ -15,5 +15,19 @@ test('ICAL parser integration test', () => {
   //   `export const events = ${JSON.stringify(events)};`
   // );
 
-  expect(JSON.stringify(events)).toBe(JSON.stringify(EVENT_DATA));
+  let i = 0;
+  events.forEach((element) => {
```

Comment:
FYI ES6 array functions, like forEach, provide an index for you so you don’t have to manually increment it. 
```suggestion
  events.forEach((element, i) => {
```

Reply:
No longer need this for each but I shall remember this info next time I use it!